src/manager.rs

```diff
@@ -69,6 +69,30 @@ impl Manager {
         }).or(Err(Error::InvalidParameter("home_id", "Manager::remove_node")))
     }
 
+    pub fn test_network(&self, home_id:u32, count:u32) {
+        unsafe {
+            extern_manager::manager_test_network(self.ptr, home_id, count);
+        }
+    }
+
+    pub fn test_network_node(&self, home_id:u32, node_id:u8, count:u32) {
+        unsafe {
+            extern_manager::manager_test_network_node(self.ptr, home_id, node_id, count);
+        }
+    }
+
+    pub fn heal_network(&self, home_id:u32, do_rr:bool) {
+        unsafe {
+            extern_manager::manager_heal_network(self.ptr, home_id, do_rr);
+        }
+    }
+
+    pub fn heal_network_node(&self, home_id:u32, node_id:u8, do_rr:bool) {
+        unsafe {
+            extern_manager::manager_heal_network_node(self.ptr, home_id, node_id, do_rr);
+        }
+    }
```

Comment:
nit: because we export `manager_heal_network` to `heal_network` in https://github.com/fxbox/openzwave-rust/pull/32/files#diff-dca5e8880d0a1a1e9e5d8a8240d3f014R26 you can use the shorter version directly eg `extern_manager::heal_network`. Same for all the methods of course.